packages/Slider/src/index.js

```diff
@@ -0,0 +1,148 @@
+import Inferno   from 'inferno'
+import Component from 'inferno-component'
+import { bind }  from 'decko'
+
+export class Slider extends Component {
+  state = {
+    focus: false,
+    keyFocus: false,
+    progress: 40
+  }
+
+  vise = (min, value, max) =>
+    Math.min(Math.max(min, value), max)
+
+  @bind
+  moveSlider({ clientX }) {
+    const { offsetLeft, clientWidth } = this.slider
+    const percentage = (clientX - offsetLeft) / clientWidth
+    const progress = this.vise(0, percentage, 1) * 100
+
+    this.setState({ progress })
+  }
+
+  @bind
+  handleMouseDown(e) {
+    this.setState({ focus: true })
+
+    this.moveSlider(e)
+  }
+
+  @bind
+  handleMouseUp(e) {
+    this.setState({ focus: false })
+  }
+
+  @bind
+  handleMouseMove(e) {
+    if(!this.state.focus) return
+
+    this.moveSlider(e)
+  }
+
+  @bind
+  handleKeyDown({ keyCode }) {
+    let { keyFocus, progress } = this.state
+
+    switch (keyCode) {
+
+      // Enter: Focs/unfocus
+      case 13:
+        this.setState({ keyFocus: !keyFocus })
+      break
+
+      // Forward: increment of 1
+      case 39:
+        if(keyFocus && progress != 100)
+          this.setState({ progress: progress+1 })
```

Comment:
The **increment** should happen even when pressing the **arrow_key_up**, simply add another case.